handler.ts

```diff
@@ -29,15 +29,16 @@ const BugsnagPluginAwsLambda = require('@bugsnag/plugin-aws-lambda')
 const {
   BUGSNAG_NOTIFIER_KEY,
   CSV_ENABLED,
-  CUSTOM_PELIAS_URL,
   GEOCODE_EARTH_URL,
   GEOCODER,
   GEOCODER_API_KEY,
   REDIS_HOST,
   REDIS_KEY,
   SECONDARY_GEOCODE_EARTH_URL,
   SECONDARY_GEOCODER,
-  SECONDARY_GEOCODER_API_KEY
+  SECONDARY_GEOCODER_API_KEY,
+  TRANSIT_BASE_URL,
+  TRANSIT_GEOCODER
```

Comment:
We really should be documenting these somewhere... "TRANSIT_GEOCODER" is not immediately very obvious to me.